### PR TITLE
[FIX] sale: Shipping Amount for Online Orders

### PR DESCRIPTION
**Pull request by simongoffin**
The price of the shipping method was always the sales price on the product of the delivery method

opw:2646684

### CODE REVIEW

File: addons/sale/data/mail_data.xml [hunk 1]
```diff
@@ -282,7 +282,7 @@
                     % if object.carrier_id.fixed_price == 0.0:
```
Comment (fmdl) [inline addons/sale/data/mail_data.xml:282]: `if object.amount_delivery == 0.0:`

**Comment by simongoffin** [comment]
Hello @Feyensv 
Is it ok for you?

**Review by Feyensv — CHANGES_REQUESTED** [review]
Nope, this is a regression...

`carrier_id` is added in `delivery`, but `amont_delivery` is only present when `website_sale_delivery` is installed...

**Comment by C3POdoo** [comment]
Dear @simongoffin,

Thank you for your contribution but the version 14.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**